tests/performance-tests/tsconfig.json

```diff
@@ -0,0 +1,29 @@
+{
+  "compilerOptions": {
+    "target": "es2017",
+    "moduleResolution": "node",
+    "module": "commonjs",
+    "noEmit": true,
+    "allowJs": true,
+    "removeComments": false,
+
```

Comment:
Just wondering why there are spaces in this file?

Reply:
I think that's just how it came from the K6 / TS template project - I've removed them now, thanks!